client/landing/jetpack-cloud/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import config from '../../config';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import detectHistoryNavigation from 'lib/detect-history-navigation';
+import initialReducer from './reducer';
+import initJetpackCloudRoutes from './routes';
+import userFactory from 'lib/user';
+import { configureReduxStore, setupBasicMiddlewares, utils } from 'boot/common';
+import { createReduxStore } from 'state';
+import { getInitialState, persistOnChange } from 'state/initial-state';
+import { setupLocale } from 'boot/locale';
+
+/**
+ * Style dependencies
+ */
+import 'assets/stylesheets/jetpack-cloud.scss';
+
+const boot = currentUser => {
+	utils();
+	getInitialState( initialReducer ).then( initialState => {
```

Comment:
Since we build our own reducer here, let's make sure to use a different redux state key, to not conflict with Calypso's state.